docker-compose.accounts.yml

```diff
@@ -0,0 +1,163 @@
+version: "3.7"
+
+x-logging: &default-logging
+  driver: json-file
+  options:
+    max-size: "10m"
+    max-file: "3"
+
+services:
+  webapp:
+    build:
+      args:
+        WITH_ACCOUNTS: 1
+
+  nginx:
+    volumes:
+      - ./docker/accounts/nginx.secure.conf:/etc/nginx/conf.extra.d/nginx.secure.conf:ro
+
+  accounts:
+    build:
+      context: ./docker/accounts
+      dockerfile: Dockerfile
+    container_name: accounts
+    restart: unless-stopped
+    logging: *default-logging
+    env_file:
+      - .env
+    environment:
+      - SKYNET_DB_HOST=mongo
+      - SKYNET_DB_PORT=27017
+      - SKYNET_DB_USER=${SKYNET_DB_USER}
+      - SKYNET_DB_PASS=${SKYNET_DB_PASS}
+      - COOKIE_DOMAIN=${COOKIE_DOMAIN}
+      - COOKIE_HASH_KEY=${COOKIE_HASH_KEY}
+      - COOKIE_ENC_KEY=${COOKIE_ENC_KEY}
+    expose:
+      - 3000
+    networks:
+      shared:
+        ipv4_address: 10.10.10.70
+    depends_on:
+      - mongo
+      - oathkeeper
```

Comment:
Looking at this got me thinking, we have a bunch of services that depend on each other in some way, do we need to describe that with `depends_on` instructions or are we fine with the services not functioning properly until all pieces are up?

We have this (`->` is "depends on"):
* accounts -> mongo
* accounts -\> oathkeeper (we can remove the dep)
* kratos -> kratos-migrate
* kratos-migrate -> cockroach (it can start without cockroach but it will wait and retry until that's up)
* kratos -> cockroach

I'm not sure the webapp really depends on anything - it can function all on its own, the only thing it really needs is `siad`. But the "please log in" functionality will be defunct until oathkeeper and accounts are up.

The longest chain seems to be:
webapp -> oathkeeper -> kratos -> kratos-migrate -> cockroach (slow to start because of the clustering)

Maybe the current setup is fine - all pieces start in parallel and only function after the entire stack is running.